.github/workflows/build.yml

```diff
@@ -31,18 +31,19 @@ jobs:
 
     steps:
       - name: Check out
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           submodules: 'recursive'
           fetch-depth: 0
 
       - name: Set up JDK 11
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
+          distribution: 'zulu'
```

Comment:
zulu was implicitly the default distro in v1. But this distribution input is now mandatory
https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md